src/azure-cli/HISTORY.rst

```diff
@@ -229,6 +229,10 @@ Release History
 * Remove unrelated exception from error message
 * Fix issues with incorrect error message "You do not have the required permissions needed to perform this operation." when blocked by network rules or AuthenticationFailed.
 
+**SQL VM**
```

Comment:
Please rebase dev branch and place your test above 2.0.78 for next release.

Reply:
Done. Thank you